### DISCUSSION TITLE
Failed partial uploads

### DISCUSSION THREAD

**Discussion by mikescandy** [immich-app/immich#13811] [Q&A]
I noticed that my upload folder was 3/4 times bigger than my actual library, and while looking at the files I bumped on a lot of mp4 files that seem like partial uploads.
I definitely found multiple partial copies of the same video.
Is there something I could do to clear up these files?

**Comment by tye-mustafa** [comment]
you should just upload each and everything from computer laptop desktop. Immich can distinguish which is file is already uploaded kind of sum check just don't interrupt the upload process this might get you caught in errors of machine learning.

the upload folder is bigger than your actual library?
the reason is each pics gets 2 additional low format pics 1. thumbnail 2. mid-size pic for faster loading on device
same for video a medium size video is additional generated by your server for faster viewing so your upload library would be 20-30% larger than your actual library totaly depends on your selected settings for image and video encoding
